File: app/src/main/java/com/verbatoria/presentation/blocked/BlockedFragment.java
package com.verbatoria.presentation.blocked;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.remnev.verbatoria.R;

/**
 * Фрагмент для отображения заблокированного состояния
 *
 * @author nikitaremnev
 */
public class BlockedFragment extends Fragment {

    public BlockedFragment() {
        // Required empty public constructor
    }

    public static BlockedFragment newInstance() {
        return new BlockedFragment();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_blocked, container, false);
    }
}

File: app/src/main/java/com/verbatoria/ui/blocked/BlockedFragment.kt
package com.verbatoria.ui.blocked

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.remnev.verbatoria.R

/**
 * @author n.remnev
 */

class BlockedFragment : Fragment() {

    companion object {

        fun createFragment(): BlockedFragment = BlockedFragment()

    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_blocked, container, false)
    }

}